crates/api/src/site/registration_applications/approve.rs

```diff
@@ -44,14 +49,20 @@ impl Perform for ApproveRegistrationApplication {
       .accepted_application(Some(data.approve))
       .build();
 
-    let approved_user_id = registration_application.local_user_id;
-    LocalUser::update(context.pool(), approved_user_id, &local_user_form).await?;
-
-    if data.approve {
-      let approved_local_user_view = LocalUserView::read(context.pool(), approved_user_id).await?;
+    let applicant_user_id = registration_application.local_user_id;
+    LocalUser::update(context.pool(), applicant_user_id, &local_user_form).await?;
 
-      if approved_local_user_view.local_user.email.is_some() {
-        send_application_approved_email(&approved_local_user_view, context.settings())?;
+    // Handle approval/denial
+    let applicant_local_user_view = LocalUserView::read(context.pool(), applicant_user_id).await?;
+    if applicant_local_user_view.local_user.email.is_some() {
```

Comment:
Again, use `if let Some(email) = applicant_local_user_view.local_user.email` and pass that into the send methods. Then theres no need for expect.

Reply:
Done.